### PR TITLE
Remove unused /frontend/public/components/operator-hub/operator-hub-community-provider-modal.tsx

### PR DESCRIPTION
**Pull request by TheRealJon**

### CODE REVIEW

**Review by spadgett — APPROVED** [review]
Thanks @TheRealJon 

/lgtm

**Comment by spadgett** [comment]
Holding until #8289 merges to unblock CI.

/hold

**Review by zherman0 — APPROVED** [review]

**Pull request merged**
